dali/python/dummy/__init__.py.in

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2017-2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import os
+
+new_path = os.path.join('https://developer.download.nvidia.com/compute/redist', '@DALI_FLAVOR_MINUS@')
+
+warning_msg = """
+###########################################################################################################
+WARNING: Package nvidia-dali@DALI_FLAVOR_MINUS@ is now deprecated. Please install nvidia-dali@DALI_FLAVOR_MINUS@-cu@CUDA_VERSION_SHORT_DIGIT_ONLY@ instead:
+pip install --extra-index-url {} nvidia-dali@DALI_FLAVOR_MINUS@-cu@CUDA_VERSION_SHORT_DIGIT_ONLY@
+For more information, go to https://docs.nvidia.com/deeplearning/dali/user-guide/docs/installation.html#id1
+###########################################################################################################
+""".format(new_path)
+
+print(warning_msg)
+
+# we need to have that defined and accessible before any other DALI import
+__version__ = '@DALI_VERSION@'
+__cuda_version__ = int('@CUDA_VERSION@'.replace('.', ''))
+__git_sha__ = '@GIT_SHA@'
+
+from . import ops
```

Comment:
Do we have this imports available in the dummy package?

Reply:
no, but the dummy package just replaces this one file in `nvidia/dali/__init__.py`. It exactly the same as in the other package plus the warning

Reply:
Sure